src/main/java/com/keyboardbuilder/enterprise/dao/BuildDAOSQL.java

```diff
@@ -0,0 +1,50 @@
+package com.keyboardbuilder.enterprise.dao;
+
+import com.keyboardbuilder.enterprise.dto.Build;
+import org.springframework.beans.factory.annotation.Autowired;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BuildDAOSQL implements IBuildDAO {
+
+    @Autowired
+    BuildRepository buildRepository;
```

Comment:
Automatic datasource for using the CrudRepository to store and retrieve build records.